src/app/legacy/containers/EpisodeList/Image.jsx

```diff
@@ -30,16 +30,31 @@ const Wrapper = styled.div`
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
     ${({ dir }) =>
       `margin-${dir === 'ltr' ? 'right' : 'left'}: ${GEL_SPACING_DBL};`}
-    ${({ isLite }) => `width: ${isLite ? '7.5rem' : '14.375rem'};`}
+    width: 14.375rem;
   }
 `;
 
+const LitePlayWrapper = withEpisodeContext(styled.div`
+  padding: ${GEL_SPACING_HLF};
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    padding: ${GEL_SPACING};
+  }
+  svg {
+    margin: 0 0 1px 0;
+    height: 0.6rem;
+    width: 0.7rem;
+    color: ${props => props.theme.palette.WHITE};
+    @media screen and (forced-colors: active) {
+      fill: linkText;
```

Comment:
Should this be the link colour? Again not quite sure what this is, but is does say styled div...

Reply:
This is the svg found on onDemandTV pages
![image](https://github.com/user-attachments/assets/da43977e-b1f7-4c18-b8f6-38fa804b3d78)

This will fill the svg with the system text link colour, so it looks right to me afaik